hs_core/tests/api/native/test_add_resource_files.py

```diff
@@ -81,3 +82,4 @@ def test_add_files(self):
         self.assertTrue(self.n1 in file_list, "file 1 has not been added")
         self.assertTrue(self.n2 in file_list, "file 2 has not been added")
         self.assertTrue(self.n3 in file_list, "file 3 has not been added")
+        delete_resource(res.short_id)
```

Comment:
I tried ```BaseResource.objects.all().delete()``` and it only delete tables but not delete files. I have not tried resource.delete() function call, if it works, I can revert it to resource.delete(), but I see calling delete_resource() has the same effect as resource.delete() assuming resource.delete() would work, right?